controllers/glance_controller.go

```diff
@@ -663,6 +667,27 @@ func (r *GlanceReconciler) reconcileNormal(ctx context.Context, instance *glance
 		instance.Status.Conditions.Set(apiCondition)
 	}
 
+	// create CronJob
```

Comment:
Maybe we should add a little more detail here to the comment, since it's not completely obvious from the code itself as to what this cron job does?

Reply:
Of course, makes sense. Doing in my next iteration

Reply:
Done

Reply:
@abays not sure it solves your concern: you ok to move forward w/ this patch?